WooCommerce/WooCommerceTests/Mocks/MockCardPresentPaymentsStoresManager.swift

```diff
@@ -94,7 +94,7 @@ final class MockCardPresentPaymentsStoresManager: DefaultStoresManager {
         case .disconnect(let onCompletion):
             onCompletion(Result.success(()))
         default:
-            fatalError("Not available")
```

Comment:
This fatalError causes a crash for JetpackSetupCoordinatorTests, so I made this silent to avoid that.